instructors.md

```diff
@@ -6,11 +6,31 @@ subtitle: Instructor's Guide
 
 ## Timing
 
-Leave about 30 minutes at the start of each workshop and another 15 mins 
-at the start of each sesson for technical difficulties like WiFi and 
-installing things (even if you asked students to install in advance, longer if 
+Leave about 30 minutes at the start of each workshop and another 15 mins
+at the start of each sesson for technical difficulties like WiFi and
+installing things (even if you asked students to install in advance, longer if
 not).
 
+
+## Setting up git in RStudio
+
+There can be difficulties linking git to RStudio depending on the operating system and the version of the operating system. To make sure git is properly installed, the learners should go to the Options window.
```

Comment:
I might change from 
"...properly installed, the learners should go to the Options window." 
to 
"... properly installed and configured, the learners should go to the Options window in the RStudio application:"